### ISSUE TITLE
Failure to compile on the PPC64LE architecture

### ISSUE THREAD

**Issue by Slime90** [obsproject/obs-studio#2301]
Platform: Talos Blackbird

PPC64LE support was added [Here](https://github.com/obsproject/obs-studio/pull/1582/commits/36a6629e6b21313437dda49675f72ee0dba60163) in this commit

OBS studio fails to compile, output is below:

```[ 30%] Building C object plugins/obs-outputs/CMakeFiles/obs-outputs.dir/ftl-stream.c.o
/home/username/src/obs-studio/plugins/linux-v4l2/v4l2-controls.c:34:2: error: #error "unknown platform, this code won't work"
   34 | #error "unknown platform, this code won't work"
      |  ^~~~~
/home/username/src/obs-studio/plugins/linux-v4l2/v4l2-controls.c: In function ‘v4l2_control_changed’:
/home/username/src/obs-studio/plugins/linux-v4l2/v4l2-controls.c:50:15: error: implicit declaration of function ‘POINTER_TO_UINT’ [-Werror=implicit-function-declaration]
   50 |  control.id = POINTER_TO_UINT(data);
      |               ^~~~~~~~~~~~~~~
[ 31%] Building C object plugins/obs-ffmpeg/CMakeFiles/obs-ffmpeg.dir/obs-ffmpeg-audio-encoders.c.o
[ 32%] Building C object plugins/obs-ffmpeg/CMakeFiles/obs-ffmpeg.dir/obs-ffmpeg-nvenc.c.o
[ 32%] Building C object plugins/obs-outputs/CMakeFiles/obs-outputs.dir/ftl-sdk/libftl/hmac/hmac.c.o
/home/username/src/obs-studio/plugins/linux-v4l2/v4l2-controls.c: In function ‘v4l2_update_controls_menu’:
/home/username/src/obs-studio/plugins/linux-v4l2/v4l2-controls.c:74:10: error: implicit declaration of function ‘UINT_TO_POINTER’ [-Werror=implicit-function-declaration]
   74 |          UINT_TO_POINTER(qctrl->id));
      |          ^~~~~~~~~~~~~~~
[ 32%] Building C object plugins/obs-outputs/CMakeFiles/obs-outputs.dir/ftl-sdk/libftl/hmac/sha2.c.o
```

**Comment by Fenrirthviti** [comment]
Looks related to #2287. Are you building from master? This fix was recently merged.

Also, in the future please make sure you are filling out the issue template and not just deleting it.

**Comment by Slime90** [comment]
I am building from master, uname -p returns the correct architecture string "ppc64le".

**Comment by FlyGoat** [comment]
Fix #2467 have been merged.
Please have a try.

**Comment by Fenrirthviti** [comment]
Fixed in master and no updates for some time, feel free to reopen if this is still an issue but it looks to be resolved.
